nodes/edge/nodes-edge-remote-workers.adoc

```diff
@@ -17,7 +17,7 @@ However, having remote worker nodes can introduce higher latency, intermittent l
 
 * *Power outage*: Because the control plane and remote worker nodes are in separate locations, a power outage at the remote location or at any point between the two can negatively impact your cluster. See xref:../../nodes/edge/nodes-edge-remote-workers.adoc#nodes-edge-remote-workers-power_nodes-edge-remote-workers[Power loss on remote worker nodes] for information on how {product-title} responds to a node losing power and for methods to diminish the impact to your cluster.
 
-* *Latency spikes or temporary reduction in throughput*: As with any network, any changes in network conditions between your cluster and the remote worker nodes can negatively impact your cluster. These types of situations are beyond the scope of this documentation.
+* *Latency spikes or temporary reduction in throughput*: As with any network, any changes in network conditions between your cluster and the remote worker nodes can negatively impact your cluster. {product-title} offers multiple _worker latency profiles_ that let you control the reaction of the cluster to latency issues. 
```

Comment:
Should this be 'cluster-to-latency' issues? This doesn't seem like a  valid adjective though.
https://www.ibm.com/docs/en/ibm-style?topic=punctuation-hyphens

Reply:
@tmalove No.  It is _the cluster's reaction to latency issues_. Which is what I had before a peer review suggestion to change to what us here!